src/index.js

```diff
@@ -67,29 +76,86 @@ export function getTagValue(list, name) {
 		return;
 	}
 
-	let jwk = JSON.parse(Buffer.from(DEPLOY_KEY, 'base64').toString('utf-8'));
+	// Throw an error if both --eth and --pol are true
+	if (argv.eth && argv.pol) {
+		console.error('Error: Cannot deploy with both ETH and POL.');
+		process.exit(1); // Exit with an error code
+	}
+
+	let jwk;
+	if (argv.pol || argv.eth) jwk = DEPLOY_KEY;
+	else {
+		jwk = JSON.parse(Buffer.from(DEPLOY_KEY, 'base64').toString('utf-8'));
+	}
+
 	try {
-		const manifestId = await TurboDeploy(argv, jwk);
-		const signer = new ArweaveSigner(jwk);
+		let signer;
+		let token;
+
+		// Creates proper signer based on wallet type.
+		switch (true) {
+			case argv.eth:
+				signer = new EthereumSigner(jwk);
+				token = 'ethereum';
+				break;
+			case argv.pol:
+				signer = new EthereumSigner(jwk);
+				token = 'pol';
+				break;
+			default:
+				signer = new ArweaveSigner(jwk);
+				token = 'arweave';
+				break;
```

Comment:
We should throw if we cannot identify the sig type instead of defaulting - if an invalid sig type is provided, we should provide feedback based on that.

Reply:
addressed

Reply:
no it wasnt